team-norms/supported_software.md

```diff
@@ -0,0 +1,25 @@
+# Supported Software
+
+Where the day-to-day running of an application needs to transition to our support team, there are specific considerations you must take when making technology choices.
+
+As an organisation we're keen to embrace new and diverse technology where there's a strong commercial justification.
+
+When you're considering a major technology choice that isn't documented here, before starting a production implementation you must seek agreement from the support team that they're able to provide BAU support.
+
+The decision to self-host a third party application must involve the support team. You can expect to be encouraged to seek SaaS alternatives.
+
+Where the application is transitioning to a customer support team for ongoing support and maintenance, you're strongly advised to consult with the customer team on any major technology decisions.
+
+## Currently Supported Software
+
+- Ruby on Rails
+- Terraform
+- Ansible
+- MySQL
```

Comment:
Wonder if we want to be more specific and say these should be AWS RDS variations only. No one wants to manage a DB these days...

Reply:
Is this already covered earlier? "The decision to self-host a third party application must involve the support team. You can expect to be encouraged to seek SaaS alternatives."